src/chrome/content/rules/Fastmail-User.FM.xml

```diff
@@ -0,0 +1,24 @@
+<!--
+          For other Fastmail coverage, see Fastmail.xml 
+
+
+		  Rewrite rule for user.fm illustrated:
+
+          http://robm.fastmail.fm.user.fm/articles/arthurs_walk.html
+                 ^^^^^^^^^^^^^^^^         ^^^^^^^^^^^^^^^^^^^^^^^^^^
+                        $1                             $2
+
+      => https://user.fm/www.robm.fastmail.fm/articles/arthurs_walk.html
+                         ^^^^^^^^^^^^^^^^^^^^ ^^^^^^^^^^^^^^^^^^^^^^^^^^
+                                  $1                      $2
+-->
+<ruleset name="Fastmail User Websites">
+	<target host="user.fm" />
+	<target host="*.user.fm" />
```

Comment:
Please add a comment explaining why this wildcard is necessary.

Reply:
Fastmail user websites
		If www.example.com was hosted on Fastmail, it would also be available 
		as http://www.example.com.user.fm/ and https://user.fm/www.example.com/